src/pages/team/index.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @next/next/no-img-element */
+import React from "react";
+import TeamHero from "../../components/team/TeamHero";
+import TeamCard from "../../components/team/TeamCard";
+import { departmentsData } from "../../components/team/team-data";
+import FooterComp from "../../components/FooterComp";
+import { DepartmentsEnum } from "components/team/team-interfaces";
+import SubscriptionForm from "components/team/SubscriptionForm";
+
+const TeamPage = () => {
+	return (
+		<div className="team">
+			<TeamHero heroTitle="Our Team" />
+
+			<main className="container-fluid team-categories">
+				<div className="strokes">
+					<img src="/images/Team/Ellipse-right.png" alt="" id="ellipse-right" />
```

Comment:
@utin-francis-peter why would you use and ID here ?

```jsx
<img src="/images/Team/Ellipse-right.png" alt="" id="ellipse-right" />
```

Reply:
To target the image and style.

Should className be used instead?